src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5764,21 +5760,6 @@ private void WriteMessageAttrib(StringBuilder stringBuilder, string elementName,
             return value;
         }
 
-        /// <summary>
-        /// There's no API to enumerate all languages an assembly is localized into, so instead
-        /// we enumerate through all the "known" cultures and attempt to load a corresponding satellite
-        /// assembly
-        /// </summary>
-        /// <returns></returns>
-        private static List<CultureInfo> GetSupportedCultures()
```

Comment:
This is interesting, I am not sure we had such code in the first place which just create a list and add the UI culture. 

I guess there were fallback mechanism before which may allow adding more cultures to the list and then get removed? 

anyway, this change looks good to me.

Reply:
> I guess there were fallback mechanism before which may allow adding more cultures to the list and then get removed?

Yeah, it used to contain this:
https://github.com/dotnet/coreclr/blob/144c15ead4c88a8c3e8840cb8b8b0a87e03019f8/src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs#L6645-L6651